### PR TITLE
setup cypress configuration

### PR DESCRIPTION
**Pull request by pelan23**
Installation and configuration of Cypress

### CODE REVIEW

File: cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```
Comment (JuanDelahozMiranda) [inline cypress/support/commands.js]: Fix EOF (append a blank line)

File: cypress/support/e2e.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (JuanDelahozMiranda) [inline cypress/support/e2e.js]: Fix EOF (append a blank line)

**Review by JuanDelahozMiranda — CHANGES_REQUESTED** [review]

File: cypress/support/commands.js (hunk 1 above)
Comment (pelan23) [reply cypress/support/commands.js]: fixed

File: cypress/support/e2e.js (hunk 2 above)
Comment (pelan23) [reply cypress/support/e2e.js]: Fixed

**Review by JuanDelahozMiranda — APPROVED** [review]

**Pull request merged**